cfme/js.py

```diff
@@ -20,8 +20,8 @@
     spinner: (!isHidden(document.getElementById("spinner_div")))
         && isHidden(document.getElementById("lightbox_div")),
     document: document.readyState,
-    autofocus: checkMiqQE('autofocus'),
-    miqQE: checkAllMiqQE()
+    autofocus: typeof(checkMiqQE) === undefined ? checkMiqQE('autofocus') : 0,
+    miqQE: typeof(checkAllMiqQE) === undefined ? checkAllMiqQE() : 0
```

Comment:
Long story short - It's javascript, nobody's gonna redefine undefined, it doesn't matter.